<launch>
<arg name="MINIBOT_UPPER_STATE" default="telebot/L_1/motion/public/minibot_state"/>
<!-- This might fit better in the launch folder of minibot mirroring package -->
<node pkg="motion" exec="driver_upper" name="driver_upper">
    <param from="$(find-pkg-share motion)/config/minibot_config.yaml"/>
    <remap from="public/upper_state" to="$(var MINIBOT_UPPER_STATE)"/>
    <remap from="private/motor_goals" to="telebot/L_1/motion/private/motor_goals_minibot"/>
</node>
<node pkg="minibot_mirroring" exec="minibot_mirror" name="minibot_mirror">
    <remap from="public/telebot_goals" to="telebot/L_2/minibot_mirror/public/telebot_goals"/>
    <remap from="public/minibot_state" to="$(var MINIBOT_UPPER_STATE)"/>
</node>
</launch>